src/Catalyst.Core.Modules.Dfs/WebApi/V0/Controllers/BlockController.cs

```diff
@@ -0,0 +1,169 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Dfs;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Catalyst.Core.Modules.Dfs.Controllers.V0;
+using Catalyst.Core.Modules.Dfs.Extensions;
+using Catalyst.Core.Modules.Dfs.WebApi.V0.Dto;
+using Lib.P2P;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using MultiFormats;
+
+namespace Catalyst.Core.Modules.Dfs.WebApi.V0.Controllers
+{
+    /// <summary>
+    ///   Manages IPFS blocks.
+    /// </summary>
+    /// <remarks>
+    ///   An IPFS Block is a byte sequence that represents an IPFS Object 
+    ///   (i.e. serialized byte buffers). It is useful to talk about them as "blocks" in Bitswap 
+    ///   and other things that do not care about what is being stored. 
+    ///   <para>
+    ///   It is also possible to store arbitrary stuff using ipfs block put/get as the API 
+    ///   does not check for proper IPFS Object formatting.
+    ///   </para>
+    ///   <note>
+    ///   This may be very good or bad, we haven't decided yet 😄
+    ///   </note>
+    /// </remarks>
+    public class BlockController : IpfsController
+    {
+        /// <summary>
+        ///   Creates a new controller.
+        /// </summary>
+        public BlockController(IDfsService dfs) : base(dfs) { }
+
+        /// <summary>
+        ///   Get the data of a block.
+        /// </summary>
+        /// <param name="arg">
+        ///   The CID of the block.
+        /// </param>
+        [HttpGet, HttpPost, Route("block/get")]
+        [Produces("application/octet-stream")]
+        public async Task<IActionResult> Get(string arg)
+        {
+            //var cid = new MultiHash(arg.FromBase32()).ToCid();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)